client/src/components/AddNewProfile.js

```diff
@@ -23,6 +24,7 @@ const AddNewProfile = () => {
   const [profileCreated, setProfileCreated] = useState(null);
   const [nationalities, setNationalities] = useState([]);
   const [groups, setGroups] = useState([]);
+  const [languages, setLanguages] = useState([]);
```

Comment:
Since the languages are constant, they should be cached globally and passed as properties where needed, rather than being re-loaded every time the user navigates to a page that uses them (see the recent changes to `ViewProfile` and how it uses nationalities, for example (similar constant data)).